react/src/components/molecules/ImagePromo/ImagePromo.stories.js

```diff
@@ -34,7 +34,7 @@ const getCommonPropsWithKnobs = () => ({
   phone: null
 });
 
-storiesOf('molecules/ImagePromo', module).addDecorator(withKnobs)
```

Comment:
Same as Q: for molecules/CalloutLink, these were grouped as two variations of the ImagePromo component